pkgs/applications/altcoins/polkadot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkgconfig
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "polkadot-${version}";
+  version = "0.2.17";
+
+  src = fetchFromGitHub {
+    owner = "paritytech";
+    repo = "substrate";
+    rev = "v0.2";
+    sha256 = "0v7g03rbml2afw0splmyjh9nqpjg0ldjw09hyc0jqd3qlhgxiiyj";
+  }; 
+
+  cargoSha256 = "0wwkaxqj2v5zach5xcqfzf6prc0gxy2v47janglp44xbxbx9xk08";
+
+  patches = [ ./patches/vendored-sources-0.2.patch ];
+
+  buildInputs = [ pkgconfig openssl openssl.dev ];
+
+  # No testing
```

Comment:
This comment is redundant; a better comment would explain *why* testing is disabled.

Reply:
So, as I see now, enabling tests doesn't broke the package build. Fix it.